docs/testplan/dual_tor/dual_tor_muxcable_test_plan.md

```diff
@@ -35,6 +35,10 @@ These are some of the vendor specific hardware capabilities which should be test
 
 The test can be executed on a single Tor testbed with proposed configurations that has a muxcable put in a port with driver code. For Tests that require some traffic validation will need to run on a testbed which can send trafic from NIC to ToR and vice-versa.
 
+The full testbed setup is shown below:
+
+![MuxCable_Hardware_testbed_setup](img/topology_for_cable_test.jpeg)
```

Comment:
Image seems broken in rendered document.

Reply:
Might be a limitation of the Github diff viewer, it looks ok on the source branch https://github.com/vdahiya12/sonic-mgmt/blob/testplan/docs/testplan/dual_tor/dual_tor_muxcable_test_plan.md

Reply:
It appears than when viewing the "rich diff" the images do not show up correctly, but if using "View file" the images display correctly.